com.unity.perception/Runtime/GroundTruth/LensDistortionCrossPipelinePass.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using Unity.Mathematics;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+
+#if HDRP_PRESENT
+    using UnityEngine.Rendering.HighDefinition;
+#elif URP_PRESENT
+    using UnityEngine.Rendering.Universal;
+#endif
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    /// Custom Pass which renders labeled images where each object labeled with a Labeling component is drawn with the
+    /// value specified by the given LabelingConfiguration.
+    /// </summary>
+    public class LensDistortionCrossPipelinePass : GroundTruthCrossPipelinePass
+    {
+        const string k_ShaderName = "Perception/LensDistortion";
+
+        //static int s_LastFrameExecuted = -1;
+
+        //Serialize the shader so that the shader asset is included in player builds when the SemanticSegmentationPass is used.
+        //Currently commented out and shaders moved to Resources folder due to serialization crashes when it is enabled.
+        //See https://fogbugz.unity3d.com/f/cases/1187378/
+        //[SerializeField]
+
+        // Lens Distortion Shader
+        private Shader m_LensDistortionShader;
+        private Material m_LensDistortionMaterial;
+
+        private bool fInitialized = false;
```

Comment:
Per our code conventions, private should be omitted where possible and private fields should start with m_. I'd recommend following this guide to get tooling support http://qatestrail.hq.unity3d.com/index.php?/runs/view/25567&group_by=cases:section_id&group_order=asc